neofs-testlib/neofs_testlib/env/templates/ir.yaml

```diff
@@ -69,7 +58,9 @@ mainnet:
 
 control:
   authorized_keys:  # List of hex-encoded 33-byte public keys that have rights to use the control service
-    - {{ public_key }}
+    {%- for public_key in public_keys %}
+    - {{ public_key }} # Hex-encoded public key
```

Comment:
Usually this stays single-key. Why do yo want multi-key here?

Reply:
I use here the same keys I used in `committee` section. Which key (from which IR node) should I use here? Or this should be some completely separate key?

Reply:
Can be done various ways, but since it's per-node, usually we set this node's key here. Then it could easily be used locally. Maybe you need to have more keys for your tests, I don't know.

Reply:
Updated. No need for multiple keys.